crawl-ref/source/fontwrapper-ft.cc

```diff
@@ -32,6 +32,24 @@
 # define dprintf(...) (void)0
 #endif
 
+class FontLibrary {
+public:
+    static FT_Library &get() {
+        static FontLibrary instance;
+        return instance.library;
+    }
+private:
+    FT_Library library;
+    FontLibrary () {
```

Comment:
tedious comment: `{` should be on a newline given our bracketing conventions (I won't comment all the cases of this but there are a bunch)